python/pyiceberg/utils/iceberg_base_model.py

```diff
@@ -38,6 +42,7 @@ class Config:
         keep_untouched = (cached_property,)
         allow_population_by_field_name = True
         frozen = True
+        json_encoders = {"Summary": test_encoder}
```

Comment:
What's happening here?

Reply:
Ah, that was some spielerij.  Instead of using a custom encoder, I ended up making the root of the Summary a Dict, which solved it in a much nicer way.

Reply:
I've removed this